org-roam.el

```diff
@@ -1625,9 +1625,10 @@ included as a candidate."
   (find-file (seq-random-elt (org-roam--list-all-files))))
 
 ;;;###autoload
-(defun org-roam-insert (&optional lowercase completions filter-fn description)
+(defun org-roam-insert (&optional type lowercase completions filter-fn description)
```

Comment:
can you place this argument at the end, and rename it to `link-type`? Not sure if other packages rely on this function.

Reply:
I did. 
I actually put it in the beginning to make keybinding to `org-roam-insert type` more convenient.